src/components/bodyImageHalfWidth.tsx

```diff
@@ -0,0 +1,52 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import Img from 'components/img';
+import { BodyImageProps as Props } from 'image';
+import { remSpace } from '@guardian/src-foundations';
+
+// ----- Setup ----- //
+
+const size = `calc(50% - ${remSpace[1]})`;
```

Comment:
Unfortunately percentages [aren't allowed](https://html.spec.whatwg.org/multipage/images.html#source-size-value) in the `sizes` attribute; this might work with `px`, `rem` or `vw` instead.

Reply:
I wonder if the output of calc works? I just tested increasing the screen size from phone to tablet and it requested two sized images `1000` and `2000`

Reply:
`calc` is valid according to that spec, it comes under the definition of "math functions".